CMakeLists.txt

```diff
@@ -556,6 +556,11 @@ endfunction()
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
     check_add_cc_flag("-std=c99")   # C99 mode
     check_add_cc_flag("-pipe")      # Avoid writing temporary files (for compiler speed)
+    if(UA_DEBUG)
+        check_add_cc_flag("-g -Og")
+    else()
+        check_add_cc_flag("-g -O2")
```

Comment:
I also would not add "-g" to any optimising build. Please leave this to debug builds only.